asmrun/Makefile

```diff
@@ -74,12 +74,12 @@ ASPPFLAGS += -DMODEL_$(MODEL)
 endif
 
 NATIVE_C_SOURCES := $(addsuffix .c, \
-  startup_aux startup main fail roots signals signals_asm misc freelist \
-  major_gc minor_gc memory alloc compare ints floats str array io extern \
-  intern hash sys parsing gc_ctrl md5 obj lexing $(UNIX_OR_WIN32) printexc \
-  callback weak compact finalise custom globroots backtrace_prim backtrace \
-  natdynlink debugger meta dynlink clambda_checks spacetime \
-  spacetime_snapshot afl bigarray)
+  startup_aux startup_native main fail_native roots_native signals \
+  signals_asm misc freelist major_gc minor_gc memory alloc compare ints \
+  floats str array io extern intern hash sys parsing gc_ctrl md5 obj \
+  lexing $(UNIX_OR_WIN32) printexc callback weak compact finalise custom \
+  globroots backtrace_prim_native backtrace natdynlink debugger meta \
+  dynlink clambda_checks spacetime_native spacetime_snapshot afl bigarray)
 
```

Comment:
Why `fail_native` but `signals_asm` ?  It could make sense to have the same `_native` suffix for all files.